src/Symfony/Component/DependencyInjection/EnvVarProcessor.php

```diff
@@ -141,15 +141,20 @@ public function getEnv($prefix, $name, \Closure $getEnv)
                 }
             }
 
+            $loaders = $this->loaders;
+            $this->loaders = new \ArrayIterator();
```

Comment:
what if an envLoader needs an other envLoader?

for instance, what's if the Consul loader proposed by @lyrixx https://github.com/symfony/symfony/pull/34295#issuecomment-551899027 needs a secret?

Reply:
That will work thanks to the loadedVars array - but first loaders cannot use latter ones.